ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml

```diff
@@ -32,7 +32,40 @@ chain:
         echo "overriding Monitor Test Failure in CSI tests"
         csi_exit_code=0
       fi
-      exit ${csi_exit_code}
+      
+      export TEST_CSI_DRIVER_MANIFEST="kubevirt-driver-rwx.yaml"
+      curl -L -o ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST} https://raw.githubusercontent.com/openshift/kubevirt-csi-driver/master/hack/test-driver-rwx.yaml
```

Comment:
I think we need to start pulling these test drivers from the release branch that matches the OCP release payload we're testing.

Today we pull directly from master, but the issue here is that RWX support is not supported in HCP KubeVIrt <= 4.15. This means this step will cause all of our currently stable periodics to fail.

Maybe something like this?

```
function ocp_version() {
    oc get clusterversion version -o jsonpath='{.status.desired.version}' | awk -F "." '{print $1"."$2}'
}

# curl this url and make sure to ignore 404 errors since older branches won't have this file. 
https://raw.githubusercontent.com/openshift/kubevirt-csi-driver/release-$(ocp_version)/hack/test-driver-rwx.yaml

```